frontend/src/lib/stores.ts

```diff
@@ -0,0 +1,7 @@
+import { writable } from "svelte/store";
```

Comment:
The currently searched stock is only meaningful on the explore page, so it would be better to have this as a variable of `/explore/+page.svelte`, that can be updated by the search component (e.g. `<Search current={currentStock} onUpdate={c => (currentStock = c)}>`).

Reply:
Noted! I agree with your point, it's just that right now I'm a bit too lazy and using the store seems like a shortcut. 
I'll keep it in mind and probably change this if I still have time after all the other stuff :)))